internal/ccsds/cuc.go

```diff
@@ -0,0 +1,47 @@
+// Package ccsds implements part of The Consultative Committee for Space Data Systems (CCSDS)
+// "Time Code Formats" (CCSDS 301.0-B-2 - CCSDS 301.0-B-4).
+package ccsds
+
+import (
+	"math"
+	"math/big"
+	"time"
+)
+
+var milliesPerSecond float64 = math.Pow10(9)

```

Comment:
This should be an integer
```suggestion
var milliesPerSecond int64 = 1e9
```